TAF/config/performance-metrics/configuration.py

```diff
@@ -22,15 +22,15 @@
 COMMAND_IMAGE = 17.60
 NOTIFICATIONS_IMAGE = 30.43
 SCHEDULER_IMAGE = 29.82
+COMMON_CONFIG_IMAGE = 0
```

Comment:
put the current image size for common config and nginx, or they will always be over the threshold

Reply:
The validation only check the current size is over than 100MB or not if the previous version size is 0.